### PR TITLE
Control: implement control minimum action to controllers

### PR DESCRIPTION
**Pull request by kevin-y-wang**

### CODE REVIEW

File: modules/control/controller/lon_controller.cc [hunk 1]
```diff
@@ -295,13 +299,15 @@ Status LonController::ComputeControlCommand(
   }
 
   if (calibration_value >= 0) {
-    throttle_cmd = std::abs(calibration_value) > throttle_deadzone
+    throttle_cmd = std::abs(calibration_value) >
+                           std::max(throttle_deadzone, throttle_minimum_action)
                        ? std::abs(calibration_value)
                        : throttle_deadzone;
```
Comment (xiaoxq) [inline modules/control/controller/lon_controller.cc]: You mean `max(std::abs(calibration_value), throttle_deadzone, throttle_minimum_action)`? The new code may result in `throttle_cmd < throttle_minimum_action` if `throttle_deadzone < throttle_minimum_action > std::abs(calibration_value)`.

Comment (Capri2014) [reply modules/control/controller/lon_controller.cc]: yep, simply choose the max of these three.

Comment (kevin-y-wang) [reply modules/control/controller/lon_controller.cc]: Discuss with Qi. Logic modified. Thanks.

File: modules/control/controller/lon_controller.cc [hunk 2]
```diff
@@ -295,15 +299,17 @@ Status LonController::ComputeControlCommand(
   }
 
   if (calibration_value >= 0) {
-    throttle_cmd = std::abs(calibration_value) > throttle_deadzone
+    throttle_cmd = std::abs(calibration_value) >
+                           std::max(throttle_deadzone, throttle_minimum_action)
                        ? std::abs(calibration_value)
-                       : throttle_deadzone;
+                       : std::max(throttle_deadzone, throttle_minimum_action);
```
Comment (xiaoxq) [inline modules/control/controller/lon_controller.cc]: Plz check out the `DRY principle`: https://github.com/ApolloAuto/apollo/pull/7514/files

Comment (kevin-y-wang) [reply modules/control/controller/lon_controller.cc]: Clean the codes according to the Dry Principle. Thanks.

File: modules/control/controller/mpc_controller.cc [hunk 3]
```diff
@@ -456,13 +460,14 @@ Status MPCController::ComputeControlCommand(
   double throttle_cmd = 0.0;
   double brake_cmd = 0.0;
   if (calibration_value >= 0) {
-    throttle_cmd = calibration_value > throttle_deadzone_ ? calibration_value
-                                                          : throttle_deadzone_;
+    throttle_cmd = calibration_value > throttle_lowerbound_
+                       ? calibration_value
+                       : throttle_lowerbound_;
```
Comment (xiaoxq) [inline modules/control/controller/mpc_controller.cc]: std::max

**Pull request merged**